Services/DataX.Config/DataX.Config.Input.EventHub/Processor/CreateEventHubConsumerGroup.cs

```diff
@@ -47,6 +47,7 @@ public CreateEventHubConsumerGroup(ConfigGenConfiguration configuration, IKeyVau
         public override async Task<FlowGuiConfig> HandleSensitiveData(FlowGuiConfig guiConfig)
         {
             var runtimeKeyVaultName = Configuration[Constants.ConfigSettingName_RuntimeKeyVaultName];
+            var uriPrefix = (Configuration[Constants.ConfigSettingName_SparkType].Length > 0 && Configuration[Constants.ConfigSettingName_SparkType] == Constants.SparkTypeDataBricks) ? Constants.PrefixSecretScope : Constants.PrefixKeyVault;
```

Comment:
If this is a common pattern, perhaps an object with this info should be created for both HDinsight and Databricks to hold these behaviors.

Reply:
done. Refacotred it in KeyVaultClient